README.md

```diff
@@ -831,6 +831,7 @@ Instructs the module to ignore a specified rule ID. Note that ignoring a rule in
 	location / {
 		access_by_lua '
 			waf:set_option("ignore_rule", 40294)
+			waf:set_option("ignore_rule", {40002, 41036})
```

Comment:
I'm not sure that reference docs are a great place to teach users Lua syntax. I'd expect someone using this tool has a bit of familiarity with the language in use. Having said that, if you want to flesh out the docs some more with examples and code comments explaining, that'd be happily accepted!